Mathlib/Algebra/Group/Hom/CompTypeclasses.lean

```diff
@@ -73,11 +73,9 @@ instance instId_comp {M N : Type*} [Monoid M] [Monoid N]
 lemma comp_inv {φ : M →* N} {ψ : N →* M} (h : Function.RightInverse φ ψ)
     {χ : M →* M} [IsId χ] :
     CompTriple φ ψ χ where
-  comp_eq := by
-    simp only [IsId.eq_id, ← DFunLike.coe_fn_eq, coe_comp, h.id]
-    rfl
+  comp_eq := by simp only [IsId.eq_id, ← DFunLike.coe_fn_eq, coe_comp, h.id, coe_id]
```

Comment:
You could also do `by simp` here

Reply:
```suggestion
  comp_eq := by simp [IsId.eq_id, ← DFunLike.coe_fn_eq, h.id]
```
seems to work, yeah. I have no opinion. Reviewer: feel free to apply or not.